### PR TITLE
Post example added

### PR DESCRIPTION
**Pull request by paulogdm**
This Pull Request ensures we have an example with POST.
Notice that we may have increased the complexity of `now.json` a little by adding methods.
Opinions?

### CODE REVIEW

**Comment by rauchg** [comment]
I think it's a great ratio of added complexity to customer value.

File: nodejs/README.md [hunk 1]
```diff
@@ -4,14 +4,54 @@ In this example we will be deploying a simple "Hello World" example with Node.js
...
+
+```js
+module.exports = (req, res) => {
+  let body = '';
+  req.on('data', piece => {
+    body = body + piece.toString();
+  });
+  req.on('end', () => {
+    res.end(body);
+  });
```
Comment (rauchg) [inline nodejs/README.md]: Can we use a little helper here? Like micro's

Comment (paulogdm) [reply nodejs/README.md]: Can you elaborate? I missed the notification...

**Comment by paulogdm** [comment]
For future reference, check our [node helpers](https://zeit.co/blog/now-node-helpers).

**Pull request closed without merging**